stable/fluent-bit/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if and .Values.metrics.enabled .Values.metrics.serviceMonitor.enabled }}
```

Comment:
Here you could also check if the CRD is installed using .Capabilities.APIVersions.Has "monitoring.coreos.com/v1"
Otherwise it looks good

Reply:
@tomahawk28 I see very few charts that are doing such a check. Is this a new good practice?

Reply:
I dont think its new but i've mostly been seeing it used recently. Its nice since it keeps it from breaking the install if there is no prometheus-operator

Reply:
Sorry for the delay. It's done